src/index.ts

```diff
@@ -76,6 +76,29 @@ export async function createFromKeyBundle(
   )
 }
 
+export async function sign(
+  clientAddress: string,
+  digest: Uint8Array,
+  keyType: string,
+  preKeyIndex: number = 0
+): Promise<Uint8Array> {
+  const signatureArray = await XMTPModule.sign(
+    clientAddress,
+    Array.from(digest),
+    keyType,
+    preKeyIndex
+  )
+  return new Uint8Array(signatureArray)
+}
+
+export async function exportPublicKeyBundle(
+  clientAddress: string
+): Promise<Uint8Array> {
```

Comment:
Not sure whats more helpful returning a bytearray or a string? I notice we return a string for the keybundle below.

Reply:
Byte array is good, since we are going to need to reinflate it to a full proto object later. And that process expects a byte array.